.github/workflows/report_quiz/README.md

```diff
@@ -0,0 +1,69 @@
+REPORT QUIZ BOT
+===
+
+[![telegram github quiz](https://github.com/osscameroon/global-github-actions/actions/workflows/report_a_quiz.yaml/badge.svg)](https://github.com/osscameroon/global-github-actions/actions/workflows/report_a_quiz.yaml)
+
+This workflow is to automate the management of the [osscameroon](https://osscameroon.com/) quiz competition.
+
+Requirements
+---
+
+- An API KEY from [QuizAPI](https://quizapi.io).
+- A bot token from [Telegram BotFather](https://core.telegram.org/bots/tutorial#introduction).
+- [Jq](https://github.com/jqlang/jq) (>=1.6)
+- [Bash](https://www.gnu.org/software/bash/)
+- [git](https://git-scm.com/)
+
+How to run this locally
+---
+
+1. Clone the repository
+
+   ```bash
+   git clone https://github.com/osscameroon/global-github-actions/
+   ```
+
+2. Move to the `report_quiz` folder
+
+   ```bash
+   cd .github/workflows/report_quiz
+   ```
+
+3. Run the following command to make a copy of the `env.example.sh` file
+
+   ```bash
+   cp env.example.sh env.sh
+   ```
+
+4. Edit the `env.sh` file with the necessary data
+
+5. Setting up your bash terminal
+
+   ```bash
+   source main.sh
+   ```
+
+6. Now to test this bot, you can use the following commands
+
+   ```bash
+   # Starts a new quiz competition.
+   . start_quiz_competition
+   # Fetch user answers of the qizzes.
+   . fetch_quiz_users_answers
+   # Stop the current quiz competition and submit resutls.
+   . stop_quiz_competition
```

Comment:
Again that doesn't look like a great way to run scripts, on a local terminal session.

Reply:
@rakici 
It's to be used in a script.
I used it directly in the github action, because I don't see the utility to create a script to run one command